src/context/trash-context.js

```diff
@@ -0,0 +1,83 @@
+import { createContext, useContext, useState, useEffect } from "react";
+import { useAuth, useNotes, useArchives } from "./context";
+import {
+  getTrashService,
+  moveNoteToTrashService,
+  moveArchiveToTrashService,
+  restoreFromTrashService,
+  deleteFromTrashService,
+} from "../services/services";
+
+const TrashContext = createContext();
+
+const TrashProvider = ({ children }) => {
+  const [trash, setTrash] = useState([]);
+  const {
+    authState: { isLoggedIn },
+  } = useAuth();
+  const { setNotes } = useNotes();
+  const { setArchives } = useArchives();
+
+  useEffect(() => {
+    if (isLoggedIn) {
+      (async () => {
+        const { data, status } = await getTrashService();
+        if (status === 200) {
+          setTrash(data.trash);
+        }
+      })();
+    } else setTrash([]);
```

Comment:
initial value is empty `[]` no need to do it again

Reply:
ok will correct it.